sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.1.0-beta.1 (Unreleased)
```

Comment:
No need to update this. please leave it as it is.

Reply:
Fixed